src/virtualship/cli/commands.py

```diff
@@ -38,23 +48,137 @@ def init(path):
     click.echo(f"Created '{config.name}' and '{schedule.name}' at {path}.")
 
 
-@click.command(
-    help="Download the relevant data specified in an expedition directory (i.e., by the expedition config)."
-)
+@click.command()
 @click.argument(
     "path",
     type=click.Path(exists=True, file_okay=False, dir_okay=True, readable=True),
 )
-def fetch(path):
-    """Entrypoint for the tool."""
-    raise NotImplementedError("Not implemented yet.")
+@click.option(
+    "--username",
+    type=str,
+    default=None,
+    help="Copernicus Marine username.",
+)
+@click.option(
+    "--password",
+    type=str,
+    default=None,
+    help="Copernicus Marine password.",
+)
+def fetch(path: str | Path, username: str | None, password: str | None) -> None:
+    """
+    Download input data for an expedition.
+
+    Entrypoint for the tool to download data based on space-time region provided in the
+    schedule file. Data is downloaded from Copernicus Marine, credentials for which can be
+    obtained via registration: https://data.marine.copernicus.eu/register . Credentials can
+    be provided on prompt, via command line arguments, or via a YAML config file. Run
+    `virtualship fetch` on a expedition for more info.
+    """
```

Comment:
> Do we also need to update the information on docs.oceanparcels.org, via changes to `README.md`? E.g. give more info on the credentials in the `virtualship fetch --help` info

oh yes, that was an item discussed yesterday. Updated - thoughts on this (which will be surfaced in the --help command)? I don't want to double up on the info, hence the `Run
    `virtualship fetch` on a expedition for more info.`. Not sure if `Credentials can
    be provided on prompt` can be worded better

Reply:
I'll update the readme once confirmed